views/file-error.html

```diff
@@ -7,7 +7,7 @@
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">{{heading}}</h1>
+      <h1 class="govuk-heading-l">{{heading}}</h1>
```

Comment:
Minor styling change here (suggested and approved by @oceannee ) just to ensure that the heading text appears on one line.